data/campaigns/Eastern_Invasion/scenarios/01_Eastern_Invasion.cfg

```diff
@@ -0,0 +1,948 @@
+# then give ground after Mal-Ravanal shows up, during the second night.
+# You need to have enough units/hp to push forward during the third day.
+#
+
+[scenario]
+    id=01_Eastern_Invasion
+    name= _ "Eastern Invasion"
+    map_file=01_Eastern_Invasion.map
+    next_scenario=02_The_Escape_Tunnel
+    turns=11
+    victory_when_enemies_defeated=no
+
+    {DEFAULT_SCHEDULE_DUSK}
+
+    {SETUP_CAMPFIRES}
+
+    {INTRO_AND_SCENARIO_MUSIC main_menu.ogg loyalists.ogg}
+    {LET_INTRO_MUSIC_FINISH}
+    [story]
+        [part]
+            # It is 625 YW. Konrad II is king, having reigned since 612.
+            story= _ "It was the thirteenth year of Konrad II’s reign when the strange occurrences in the Estmark Hills on the eastern border of Wesnoth began."
+            background=story/S01-1.jpg
+        [/part]
+        [part]
+            story= _ "Cattle and beasts of burden were found dead in the fields and villagers vanished from their houses with nary a clue nor sign of struggle. At first, the settlers suspected raiders from the great desert, but when they sent scouts east they found nothing but a small mountain range and miles of swamp."
+            background=story/S01-2.jpg
+        [/part]
+        [part]
+            story= _ "The disappearances did not stop and every day the terror grew greater. Eventually, the people living in the eastern villages sent a messenger to the King, asking for help. The messenger rode hard for many days and nights and finally reached the city of Weldyn."
+            background=story/S01-3.jpg
+        [/part]
+        [part]
+            story= _ "When the King received news of the problem, his advisor Dacyn had much to say. Apparently, these attacks bore much similarity to ones that had previously occurred on the far southern border where undead had been the eventual culprit. Dacyn outlined a plan to combat these intrusions."
+            background=story/S01-4.jpg
+        [/part]
+        [part]
+            # The River Guard posts had been built in 470 YW. They were
+            # abandoned in 544 YW; the wave of colonization had begun around
+            # 530 YW.
+            story= _ "In the days of King Garard I, two strong points had been built along the near bank of the Weldyn, south of Soradoc, to stop bandits and orcish raiders out of the Estmarks from entering Wesnoth. In later years, the River Guard posts had been abandoned as colonists spread into the Estmarks and the orcs were driven in retreat north of the Great River."
+            background=story/S01-5.jpg
+        [/part]
+        [part]
+            story= _ "With unknown enemies pressing Wesnoth from the East, Dacyn pressed Konrad II to re-build and re-man the River Guard outposts, and use them to keep the Crown’s eye and hand firmly on the settled country to either side of the lower Weldyn. He sent two of the most promising young officers to them."
```

Comment:
> either side of the lower Weldyn

Is 'Lower Weldyn' supposed to be a famous location? 'the lower Weldyn' sounds odd to me.

Reply:
Oh I see. This is referring to the River Weldyn, not the city Weldyn. Perhaps a `#po` note?